Makefile

```diff
@@ -0,0 +1,65 @@
+TARGET = halloy
```

Comment:
Do we plan on having linux commands in the Makefile ?

Reply:
I think it make sense to have it all there - think Mac should be the most complicated because .App and .Dmg?

Reply:
Do you plan on using the makefile for anything outside of CI? I would never use it normally for just building / testing the app. Maybe we just make this a script `build-macos` and throw it in `scripts` folder?

Reply:
Just feel like if ppl see `Makefile` in the root of our repo, they'll expect it to work on Linux too and I don't really want to use `Makefile` for that